src/GitUtils.ps1

```diff
@@ -517,7 +517,11 @@ function Find-Ssh($program = 'ssh-agent') {
 }
 
 # Loosely based on bash script from http://help.github.com/ssh-key-passphrases/
-function Start-SshAgent([switch]$Quiet, [string]$StartupType = 'Manual') {
+function Start-SshAgent(
+    [switch]$Quiet,
+
+    [ValidateSet("Automatic", "Boot", "Disabled", "Manual", "System")]
+    [string]$StartupType = 'Manual') {
```

Comment:
When using parameter attributes I would recommend using a param block e.g.:
```
 function Start-SshAgent {
    param(
        [Parameter(Position = 0)]
        [ValidateSet("Automatic", "Boot", "Disabled", "Manual", "System")]
        [string]
        StartupType = "Manual",

        [Parameter()]
        [switch]
        $Quiet
    )
    ...
```
Not required but it is more "canonical" PowerShell.  :-)

Reply:
@rkeithhill Ah! Thanks, I didn't know that. That's the c# developer in me ;) I'll change that now.

Reply:
Done!